codes/quantum/qudits/qudit_stabilizer.yml

```diff
@@ -8,20 +8,20 @@ physical: qudits
 logical: qudits
 
 name: '\(\mathbb{Z}_q\)-qudit stabilizer code'
-#introduced: '\cite{arXiv:quant-ph/9705052}'
+introduced: '\cite{arXiv:quant-ph/9705052}'
 
-description: 'An \(((n,p^k,d))_q\) (with \(q\) not necessarily prime) stabilizer code is denoted as \([[n,k]]_q\) or \([[n,k,d]]_q\), where \(d\) is the code''s distance. Stub.'
-#Logical subspace is the joint \(+1\) eigenspace of a set of \(p^{n-k}\) commuting qudit Pauli operators forming the code''s stabilizer group, which cannot contain \(-I\).
-#The distance is the minimum weight of an error that implements a nontrivial logical operation on the code.
-#Normalizer \(N(S)\) of the stabilizer \(S\) is the set that contains all operators that commutes with \(S\).
-#The stabilizer code can correct errors \(\varepsilon \in P_n(p)\) if and only if \(\hat{E}^\dagger \hat{F} \notin \hat{N}(S)\setminus \hat{S}\) for all \(E,F \in \varepsilon\).'
-# Defined only for prime-power qudits for now
-# Galois qudit codes have lifted stabiilizers, generally k=n-r/m
+description: 'An \(((n,q^k,d))_q\) (with \(q\gt 1\)) stabilizer code is denoted as \([[n,k]]_q\) or \([[n,k,d]]_q\), where \(d\) is the code''s distance.
+The number of stabilizer generators  equals to \( q^{n-k}\), which is proven in \cite{arXiv:1101.1519}.
+Normalizer \(N(S)\) of the stabilizer \(S\) is the set that contains all operators that commute with \(S\).
```

Comment:
Is this true for composite-q qudits? I know this is so for q=prime, see Thm. 8.6 in Gottesman's book. The reason for the complication is that Pauli matrices have different orders for non-prime q. If you're not sure this holds for composite q, let's move this to code_id: prime_qudit_stabilizer.

Reply:
You are right, I agree this is not true for composite-q.